tools/build_config/Jenkinsfile

```diff
@@ -1,18 +1,40 @@
 #!groovy
 
+def getRepoURL() {
+  sh "git config --get remote.origin.url > .git/remote-url"
+  return readFile(".git/remote-url").trim()
+}
+
+def getCommitSha() {
+  sh "git rev-parse HEAD > .git/current-commit"
+  return readFile(".git/current-commit").trim()
+}
+
 def post_github_status(String state, String message) {
-    withCredentials([string(credentialsId: 'GitHub_API_Token',
-            variable: 'api_token')]) {
-        sh """
-            curl -H "Authorization: token ${api_token}" \
-                --request POST \
-                --data '{"state": "${state}", \
-                         "description": "${message}", \
-                         "target_url": "$BUILD_URL", \
-                         "context": "$JOB_BASE_NAME"}' \
-                $PR_STATUSES_URL > /dev/null
-        """
-            }
+    // Non-PR builds will not set PR_STATUSES_URL - in which case we do not
+    // want to post any statuses to Git
+    if (env.PR_COMMIT_SHA) {
+        def repo_url = getRepoURL()
+        def commit_sha = env.PR_COMMIT_SHA
+
+    } else {
+        def repo_url = getRepoURL()
+        def commit_sha = getCommitSha()
```

Comment:
since `repo_url` is the same in both, can you replace this if with an inline if? 
```
def commit_sha = (env.PR_COMMIT_SHA) ?  env.PR_COMMIT_SHA : getCommitSha()
```